src/Api/ProductDraftUuidApi.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Pim\ApiClient\Api;
+
+use Akeneo\Pim\ApiClient\Client\ResourceClientInterface;
+use Akeneo\Pim\ApiClient\Pagination\PageFactoryInterface;
+use Akeneo\Pim\ApiClient\Pagination\ResourceCursorFactoryInterface;
+
+/**
+ * @copyright 2022 Akeneo SAS (https://www.akeneo.com)
+ * @license https://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+class ProductDraftUuidApi implements ProductDraftUuidApiInterface
+{
+    const PRODUCT_DRAFT_UUID_URI = '/api/rest/v1/products-uuid/%s/draft';
+    const PRODUCT_PROPOSAL_UUID_URI = '/api/rest/v1/products-uuid/%s/proposal';
+
+    /** @var ResourceClientInterface */
+    protected $resourceClient;
```

Comment:
```suggestion
    protected ResourceClientInterface $resourceClient;
```
And remove comments (do the same for all attributes)

Reply:
done

Reply:
refactored